### PR TITLE
installer

### PR DESCRIPTION
**Pull request by tomirons**
convers the stubs into actual policies

### CODE REVIEW

File: src/Console/ForumCommand.php [hunk 1]
```diff
@@ -11,7 +11,7 @@ class ForumCommand extends Command
      *
      * @var string
      */
-    protected $signature = 'forum:install';
+    protected $signature = 'laravel-forum:install';
```
Comment (Dylan-DPC) [inline src/Console/ForumCommand.php]: Why? I feel forum should be better.

Comment (AndreasElia) [inline src/Console/ForumCommand.php]: I can live with either, your call @tomirons

Comment (NowNewStart) [inline src/Console/ForumCommand.php]: I agree with forum:install too tho, I don't believe the user installs more than one forum package.

Comment (Dylan-DPC) [inline src/Console/ForumCommand.php]: A signature should be something that user can easily enter and doesn't have to alias

Comment (Riari) [inline src/Console/ForumCommand.php]: Yeah, no need for `laravel-forum` here. Look at other Artisan commands - their namespaces are never prefixed like that.

**Comment by AndreasElia** [comment]
Could you rename the command to `InstallCommand` please? I feel it would make more sense over `ForumCommand`.

File: src/Console/ForumCommand.php (hunk 1 above)
Comment (Dylan-DPC) [inline src/Console/ForumCommand.php]: it's about the namespace not the command. There is a good chance of having other commands under the forum namespace

Comment (Riari) [inline src/Console/ForumCommand.php]: I highly doubt that.

**Pull request merged**